fswatch/src/windows/CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+set(LIBRARY_NAME "getopt")
```

Comment:
I understand why you're creating multiple libraries etc., but it's odd, if not wrong, that the GNU gettext files are inside a directory called windows.